types/src/keyless/bn254_circom.rs

```diff
@@ -62,6 +62,18 @@ impl G1Bytes {
         Self::new_from_vec(bytes)
     }
 
+    // TODO(keyless) Shouldn't this return a result rather than unwrap()ing?
+    pub fn from_hex(hex: &str) -> Self {
```

Comment:
Remove these functions and use them directly in the Serialize/Deserialize impl block